README.md

```diff
@@ -46,7 +46,17 @@ class RecordLoader < GraphQL::Batch::Loader
 end
 ```
 
-Use lazy execution and instrumentation by `GraphQL::Batch` in your schema
+Use `GraphQL::Batch` as a plugin in your schema (for graphql > `1.5.0`).
+
+```ruby
+MySchema = GraphQL::Schema.define do
+  query MyQueryType
+
+  use GraphQL::Batch
+end
+```
+
+Or, use lazy execution and instrumentation by `GraphQL::Batch` in your schema.
```

Comment:
For older versions of graphql-ruby they should just use `GraphQL::Batch.use(self)` instead of `use GraphQL::Batch`